pandas/core/series.py

```diff
@@ -3432,7 +3432,7 @@ def explode(self) -> "Series":
 
     def unstack(self, level=-1, fill_value=None):
         """
-        Unstack, a.k.a. pivot, Series with MultiIndex to produce DataFrame.
+        Unstack, a.k.a pivot, Series with MultiIndex to produce DataFrame.
```

Comment:
Maybe just reword this to use "or" instead of "a.k.a.", or wite it in full? "also known as"

Reply:
Alright! Will do that